kalite/distributed/templates/distributed/base.html

```diff
@@ -302,6 +291,22 @@
                                 </li>
                               </ul>
                             </li>
+                            <!-- Language selection -->
+                            {% if request.session.facility_user.is_teacher %}
+                            <li class="teacher-only">
+                            {% else %}
+                            <li class="not-admin-only">
+                            {% endif %}
+                              {% if num_language_choices > 1 %}
+                              <select id="language_selector">
+                              {% for lang_code, lang_meta in language_choices.iteritems %}
+                                <option value="{{ lang_code }}" {% if lang_code == current_language %}selected{% endif %}>
+                                    {{ lang_code|get_language_name }}
+                                </option>
+                              {% endfor %}
+                              </select>
+                              {% endif %}
```

Comment:
Yikes, thanks for noticing, in fact I might just as well do the `if` on the class.